applications/MeshingApplication/custom_processes/mmg_process.h

```diff
@@ -496,11 +496,11 @@ class KRATOS_API(MESHING_APPLICATION) MmgProcess
     ///@name Un accessible methods
     ///@{
 
-//     /// Assignment operator.
-//     MmgProcess& operator=(MmgProcess const& rOther);
+    /// Assignment operator.
```

Comment:
When I commented this?

Reply:
Looks lime in #385 O.o, maybe while moving files?